src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,6 @@
+.catalog {
+  display: flex;
+  justify-content: center;
```

Comment:
![Screenshot_5](https://user-images.githubusercontent.com/104390840/207868535-b7a5dcd5-a3f3-48c9-8c75-15ce7ed5e6ed.png)
It have to work with other number of cards

Reply:
haven't fixed